app/views/items/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_with url: merchant_items_path(@merchant.id), method: :post, local: true do |form|  %>
```

Comment:
This could be an opportunity to use the form_with model from class, but not a necessary change at this point unless you're looking to practice it.

Reply:
Glad you mentioned it, will definitely get some practice in!